frontend/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -41,6 +42,7 @@ const TooltipWrapper = ({
   // tipCustomClass,
   children,
   tipContent,
+  tipOffset = 5,
```

Comment:
making this offset the default for all TooltipWrapper worries me a bit. Lets just make sure that the tooltip wrapper looks ok with this new default set.

Also just curious why we need this to be dynamic. I can see it 14 in `VersionCell`, but just curious why we want to change it here?

Reply:
Thanks for that consideration Gabe, agreed we don't want to go about changing app-wide defaults without careful consideration. This is actually just moving the definition of this default value from line 85 to here, so shouldn't change the resulting UI.

For the `VersionCell` instance, this is what makes it render the same distance from the hover element as the previous `ReactTooltip`. Slightly different than the non-arrow + underline version.